### PR TITLE
Update description for docker plugin inspect

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Update description for docker plugin inspect  which is inconsistent with the latest code.

Signed-off-by: yuexiao-wang wang.yuexiao@zte.com.cn

### CODE REVIEW

File: docs/reference/commandline/plugin_inspect.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ parent = "smn_cli"
 # plugin inspect (experimental)
 
 ```markdown
-Usage:  docker plugin inspect PLUGIN
+Usage:  docker plugin [OPTIONS] PLUGIN [PLUGIN...]
```
Comment (vdemeester) [inline docs/reference/commandline/plugin_inspect.md]: I think you missed adding back `inspect` 👼

Comment (yuexiao-wang) [inline docs/reference/commandline/plugin_inspect.md]: @vdemeester PTAL
Sorry， It is copy-paste mistake.
I have updated this PR.
Thanks!

**Comment by vdemeester** [comment]
LGTM 🐸
/cc @sfsmithcha @thaJeztah @SvenDowideit

**Comment by SvenDowideit** [comment]
LGTM - really should re-script this stuff to auto update again.

**Comment by vdemeester** [comment]
@SvenDowideit yep :stuck_out_tongue_closed_eyes:

**Pull request merged**